src/main/java/uk/gov/pay/ledger/transaction/TransactionSearchParams.java

```diff
@@ -0,0 +1,180 @@
+package uk.gov.pay.ledger.transaction;
```

Comment:
please can you put each of your new classes in an appropriate sub package, or even put everything to do with search in a new package